File: vm/beginner/l2.5/power.py
#power(x, y).    In power.py write the function power(x, y) which computes x to the y power if y is an 
# integer greater than or equal to 0. 
def power(x,y):
    p = 1
    for i in range(y):
        p = p*x
    return p
print(power(2,5))    

def power(x,y):
    return x**y
print(power(2,5))    

File: vmc/beginner/l2.5/power.c
#include<stdio.h>
int power(int x,int y)
{
    int p = 1;
    for (int i = 0; i < y; i++)
    {
        p *= x;
    }
    return p;
}
int main()
{
    printf("%d \n",power(5,0));
    printf("%d \n",power(5,1));
    printf("%d \n",power(5,2));
    printf("%d \n",power(2,5));
    return 0;
}
